Refresh.GameServer/Database/GameDatabaseProvider.cs

```diff
@@ -554,7 +568,7 @@ protected override void Migrate(Migration migration, ulong oldVersion)
             migration.NewRealm.RemoveRange(migration.NewRealm.All<GameSubmittedScore>().Where(s => s.Level == null));
 
         // fuck realm.
-        if (oldVersion < 142)
+        if (oldVersion < 152)
```

Comment:
Is it actually fixed now?

Reply:
forgot about this entirely, i was just hotfixing it locally so i could continue testing, i'll double check the code responsible for causing the issue, its a *very* annoying bug, that i *thought* i fixed

Reply:
okay, i double checked the code, i have no idea why the bug still happened, maybe something weird with my dev package? im not sure

see the relevant code [here](https://github.com/Beyley/Refresh/blob/84c3e88b000b63030c38d2fe0a5429fc9904e79e/Refresh.GameServer/Database/GameDatabaseContext.Users.cs#L332)

Reply:
Made #650 as a note.